src/lld/dpdk/Makefile.am

```diff
@@ -1,8 +1,13 @@
 all:
 	$(shell ./apply_patch.sh > /dev/null 2>&1)
 	stat dpdk_src/build > /dev/null 2>&1 || \
-	(cd dpdk_src && meson -Dprefix=$(prefix) build && \
-	cd build && ninja -j48 && ninja install)
+	(cd dpdk_src && meson -Dcpu_instruction_set=generic -Dprefix=$(prefix) build && \
```

Comment:
These changes apply for all the builds including real hardware. I feel it is better to introduce another build option and utilize it specially for vagrant builds instead of making it generic.

Reply:
Can you provide some guidance on how we might do that? I would appreciate any help to either take over this PR or guide me here. We'll need to expose this up to the IPDK level as well (e.g. P4-OVS) so it's configurable from that build level.